guides/doc-Managing_Organizations_and_Locations/master.adoc

```diff
@@ -4,13 +4,14 @@ include::common/header.adoc[]
 
 = {ManagingOrganizationsLocationsDocTitle}
 
-// This guide is not ready for stable releases
-// Satellite publishes this documentation elsewhere
-ifdef::HideDocumentOnStable,satellite[]
-include::common/modules/con_guide-not-ready.adoc[leveloffset=+1]
+ifdef::satellite[]
+{Project} documentation includes the content in {AdministeringDocURL}[{AdministeringDocTitle}]:
+
+* {AdministeringDocURL}Managing_Organizations_admin[Managing organizations]
+* {AdministeringDocURL}Managing_Locations_admin[Managing locations]
```

Comment:
```suggestion
{Project} documentation includes the content in {AdministeringDocURL}[_{AdministeringDocTitle}_].
```
I think this is enough. And it avoids the risk of the links breaking if the chapters move.